api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -180,6 +180,33 @@ def test_unregistered_contributors_show_up_as_name_associated_with_project(self)
         assert_equal(res.json['data'][1]['embeds']['users']['data']['attributes']['full_name'], 'Robert Jackson')
         assert_equal(res.json['data'][1]['attributes'].get('unregistered_contributor'), 'Bob Jackson')
 
+    def test_contributors_order_is_the_same_over_multiple_requests(self):
+        self.public_project.add_unregistered_contributor(
+            'Robert Jackson',
+            'robert@gmail.com',
+            auth=Auth(self.user),
+            save=True
+        )
+
+        for i in range(0,10):
+            new_user = AuthUserFactory()
```

Comment:
Minor: use `UserFactory` if you don't need to use basic auth in the tests. `AuthUserFactory` instances take longer to instantiate because they bcrypt passwords.